### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by vcarazo**
ms.date must be updated

### CODE REVIEW

**Comment by Jak-MS** [comment]
@vcarazo - you can't set a date in the future.

@jtarquino @maggiesMSFT 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by maggiesMSFT** [comment]
Hi, @vcarazo and @jtarquino -- the only change I see is to set the date in the future, which doesn't make sense. I'll sign off when there are actual content changes.

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by vcarazo** [comment]
This is the update date I see on this page:

Article · **07/02/2024** · 24 contributors

I dont know the format of this date but 7th feb 2024 or 2nd july 2024 are wrong dates because on September 26, 2024 this page was updated anouncing the release of power bi report server september 2024.

**Comment by Court72** [comment]
> ... on September 26, 2024 this page was updated anouncing the release of power bi report server september 2024.

Should the date be set to **09/26/2024** then?
image

File: powerbi-docs/report-server/changelog.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.reviewer: maggies
 ms.service: powerbi
 ms.subservice: powerbi-report-server
 ms.topic: conceptual
-ms.date: 07/02/2024
+ms.date: 10/26/2024
```
Comment (Court72) [inline powerbi-docs/report-server/changelog.md:10]:
```suggestion
ms.date: 09/26/2024
```

**Pull request merged**